<launch>
    <node pkg="formation_controller" type="formation_controller_node" name="formation_flt" output="screen">
        <param name="agent_id" value="uav0" />
        <param name="uav_id" value="0" />
        <param name="publish_rate" value="20" />
    </node>

    <node pkg="formation_controller" type="formation_controller_node" name="formation_flt1" output="screen">
        <param name="agent_id" value="uav1" />
        <param name="uav_id" value="1" />
        <param name="publish_rate" value="20" />
    </node>

    <node pkg="formation_controller" type="formation_controller_node" name="formation_flt2" output="screen">
        <param name="agent_id" value="uav2" />
        <param name="uav_id" value="2" />
        <param name="publish_rate" value="20" />
    </node>

    <node pkg="formation_controller" type="formation_controller_node" name="formation_flt3" output="screen">
        <param name="agent_id" value="uav3" />
        <param name="uav_id" value="3" />
        <param name="publish_rate" value="20" />
    </node>

    <node pkg="formation_controller" type="formation_controller_node" name="formation_flt4" output="screen">
        <param name="agent_id" value="uav4" />
        <param name="uav_id" value="4" />
        <param name="publish_rate" value="20" />
    </node>

</launch>